plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -294,6 +297,10 @@ public Instance startService(String namespace,
 
             docker.startContainer(StartContainerParams.create(container));
 
+            if (!Strings.isNullOrEmpty(getUser())) {
+                changeWorkingDirOwner(container, machineLogger);
```

Comment:
Sorry I misunderstood it a bit. In that case main process will run under new user but files on container's FS will be under old user. It can raise problem with that container. WDYT?